### PR TITLE
Add database

### LINKED ISSUE #20: Make database
A database will make handling data related to the scoreboard easier.

It could also be used for saving:
- Events
- Timeline data
- ribbons

### PR DESCRIPTION
**Pull request by EikHR**
Closes #20

### CODE REVIEW

**Review by ivarnakken — APPROVED** [review]
😍

**Review by norbye — APPROVED** [review]
LGTM:)

File: pages/timeline/index.tsx [hunk 1]
```diff
@@ -2,11 +2,25 @@ import type { NextPage } from "next";
...
-import timelineEvents from "../../data/timeline.json";
 import NavBar from "../../components/NavBar";
 import Header from "../../components/Header";
+import useSWR from "swr";
+import { TimelineEvent } from "models/schema";
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
 
 const TimelinePage: NextPage = () => {
+  const { data, error } = useSWR<TimelineEvent[], unknown>(
```
Comment (juniwbjerde) [inline pages/timeline/index.tsx:13]: Is there a reason you've put type=`unknown` here, or could you find out the correct type for `error` and use this instead?

File: pages/api/timelineEvents.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import mongoose from "mongoose";
+import { NextApiRequest, NextApiResponse } from "next";
+import { TimelineEvent, TimelineEventModel } from "../../models/schema";
```
Comment (LudvigHz) [inline pages/api/timelineEvents.ts:3]: absolute import pls

**Review by Arashfa0301 — APPROVED** [review]
LGTM

**Pull request merged**